src/modals/SettingsModal/index.tsx

```diff
@@ -47,6 +52,12 @@ export const settingsModalTabs: SettingsTabItem[] = [
   { value: 'devtools', label: i18next.t('Developer tools') }
 ]
 
+const socialMediaList = {
+  twitter: { url: 'https://twitter.com/alephium', src: twitterLogo },
+  discord: { url: 'https://discord.gg/XsGpZ5VDTM', src: discordLogo },
+  github: { url: 'https://github.com/alephium', src: githubLogo }
```

Comment:
Let's add these links in `src/utils/links.ts`

Reply:
We might also want to use the `www.alephium.org/discord` link as a link to Discord.

Reply:
Done